src/main/java/net/javadiscord/javabot/systems/help/HelpListener.java

```diff
@@ -208,15 +243,27 @@ private void handleHelpThanksInteraction(@NotNull ButtonInteractionEvent event,
 		ThreadChannel post = manager.getPostThread();
 		HelpConfig config = botConfig.get(event.getGuild()).getHelpConfig();
 		if (event.getUser().getIdLong() != post.getOwnerIdLong()) {
-			Responses.warning(event, "Sorry, only the person who reserved this channel can thank users.").queue();
+			Responses.warning(
+							event,
+							"Sorry, only the person who reserved this channel can thank users."
+					)
+					.queue();
 			return;
 		}
 		switch (id[2]) {
 			case "done" -> handleThanksCloseButton(event, manager, post);
 			case "cancel" -> event.getMessage().delete().queue();
 			default -> {
-				List<Button> thankButtons = event.getMessage().getButtons().stream().filter(b -> b.getId() != null && !ComponentIdBuilder.split(b.getId())[2].equals("done") && !ComponentIdBuilder.split(b.getId())[2].equals("cancel")).toList();
-				if (thankButtons.stream().filter(Button::isDisabled).count() == thankButtons.size() - 1) {
+				List<Button> thankButtons = event.getMessage()
+						.getButtons()
+						.stream()
+						.filter(b -> b.getId() != null &&
+								!ComponentIdBuilder.split(b.getId())[2].equals("done") &&
+								!ComponentIdBuilder.split(b.getId())[2].equals("cancel"))
+						.toList();
+				if (thankButtons.stream()
+						.filter(Button::isDisabled)
+						.count() == thankButtons.size() - 1) {
```

Comment:
In this case, it _might_ make more sense to put it in a single line or to add an additional line break after the `==`.